docs/dev_tools/build_profiles.md

```diff
@@ -10,6 +10,7 @@ mbed OS 5 supports three primary build profiles: *develop*, *debug* and *release
 * Largest and slowest profile.
 * Full error information. For example, asserts have file name and line number.
 * Easy to step through code with a debugger.
+* Sleep is disabled.
```

Comment:
To eliminate passive voice, I recommend changing this to "Disabled sleep" or "Disabled sleep mode". Would that keep the same meaning?